shuttle.go

```diff
@@ -116,7 +117,15 @@ func (cm *ContentManager) registerShuttleConnection(handle string, hello *drpc.H
 var ErrNilParams = fmt.Errorf("shuttle message had nil params")
 
 func (cm *ContentManager) processShuttleMessage(handle string, msg *drpc.Message) error {
-	ctx, span := cm.tracer.Start(context.TODO(), "processShuttleMessage")
+	ctx := context.TODO()
+
+	// if the message contains a trace continue it here.
+	if msg.HasTraceCarrier() {
```

Comment:
Here is where Estuary will continue a trace if one is present in the message received from the shuttle.